### PR TITLE
User balance

### PR DESCRIPTION
**Pull request by devton**
## why?

Currently when project finish we transfer pledged amount to the project owner bank_account given on project subscription process, the problem is that we need to get and check on the gateway tha bank account data.

This PR move the bank account to the final process and adds a balance/withdraw operations:
When project finish the pledged amount goes to user balance, the bank account is necessary as the final step when user calls withdraw

### CODE REVIEW

**Comment by mikesmayer** [comment]
@luisofribeiro i might be misunderstanding this -- but it looks like this branch hasn't been merged, so I should take that as mean #2524 isn't really complete just yet?

(no pressure, just want to make sure I'm following the development process properly)

**Comment by luisofribeiro** [comment]
Exact @mikesmayer 

We are expecting to merge next week.  Sorry for the confusion.

**Comment by mikesmayer** [comment]
no worries -- thanks man! keep up the good work

**Comment by mikesmayer** [comment]
hi @luisofribeiro, is there anything I can do help you guys finish up the last bits on this so we can merge this PR?

**Comment by devton** [comment]
hello @mikesmayer we are just waiting the current code goes to production to merge this branch ;)

File: app/models/balance_transaction.rb [hunk 1]
```diff
@@ -1,9 +1,60 @@
 class BalanceTransaction < ActiveRecord::Base
+  EVENT_NAMES = %w(
+    catarse_contribution_fee
+    project_contribution_confirmed_after_finished
+    balance_transfer_project
+    balance_transfer_request
+    balance_transfer_error
+    transfered_project_pledged
```
Comment (adrianob) [inline app/models/balance_transaction.rb]: Should be 'transferred'.

File: config/locales/catarse_bootstrap/views/projects/edit.pt.yml [hunk 2]
```diff
@@ -321,8 +321,8 @@ pt:
         href: '#confirm_account'
         icon: 'https://daks2k3a4ib2z.cloudfront.net/54b440b85608e3f4389db387/56c3508dbfc50aef5095c66a_finish-icon.png'
         title: 'Parabéns!'
-        text: 'Seu projeto foi finalizado com sucesso e você tem <span class="fontweight-semibold">R$ %{total_amount}</span> para receber (<a id="tax_link" class="alt-link">veja extrato</a>)! Para efetuarmos a transferência, precisamos que você nos confirme seus dados bancários.'
-        cta: 'Confirmar dados bancários'
+        text: 'Seu projeto foi finalizado com sucesso e ja foi transferido <span class="fontweight-semibold">R$ %{total_amount}</span> para receber (<a id="tax_link" class="alt-link">veja extrato</a>) em seu saldo!'
```
Comment (adrianob) [inline config/locales/catarse_bootstrap/views/projects/edit.pt.yml]: Another typo in 'ja'.

**Comment by sushant12** [comment]
When will this be merged into master?

**Comment by vicnicius** [comment]
This week probably, @sushant12. We deployed `develop` on the end of the week, so we are just going to wait a bit, fix some issues if they occur and then we should be fine to merge develop into master.

**Comment by luisofribeiro** [comment]
@sushant12 @mikesmayer  It's already merged into master. FYI 
https://github.com/catarse/catarse/pull/2665

**Pull request merged**